tests/detectors/huggingface/test_metrics.py

```
@@ -0,0 +1,104 @@
+
+class TestMetrics:
+    @pytest.fixture
+    def client(self):
+        current_dir = os.path.dirname(__file__)
+        parent_dir = os.path.dirname(os.path.dirname(current_dir))
+        os.environ["MODEL_DIR"] = os.path.join(parent_dir, "dummy_models", "bert/BertForSequenceClassification")
+
+        detector = Detector()
+
+        # patch the model to allow for control over detections - long messages will flag
+        def detection_fn(*args, **kwargs):
```

Ruff (0.14.1)

Unused function argument: `args`

(ARG001)